content/articles/intro-to-emberjs/index.md

```diff
@@ -0,0 +1,155 @@
+### Use cases
+EmberJS is a good choice for apps that want to have one or both of the following characteristics:
+
+1. Single-page apps, such as native-like web apps and progressive web apps (PWAs)
+    1. Ember performs best when it is used to build your complete application's front end..
+
+2. Improving technology stack cohesion across many teams
+    1. "Best practices" supported by the community enable speedier long-term development.
+
+    2. Ember features well-defined norms that are beneficial for enforcing uniformity and assisting team members in swiftly catching up.
+### Installing the Ember tooling
+Ember builds and scaffolds sections of your application using a command-line interface (CLI) tool.
+
+1.  Before you can install `ember-cli`, you'll need to have node and npm installed. If you don't already have node and npm installed, go here to learn how to do so.
+
+2.  To install `ember-cli`, put the following into your terminal:
+
+  
+         npm install -g ember-cli
+
+The ember program, which is used to construct, build, develop, test, and scaffold your application, is provided via this tool in your terminal (run ember —help for a full list of commands and their options).
+
+3. 	To make a new application, type the following into your terminal. Inside the current directory, a new directory named todomvc is created, which contains the scaffolding for a new Ember app. Make sure you've gone to the correct spot in the terminal before running it.
+
+
+         npm install -g ember-cli
+
+The ember program, which is used to construct, build, develop, test, and scaffold your application, is provided via this tool in your terminal (run ember —help for a full list of commands and their options).
+
+  	  ember new todomvc
+   Or, on Windows:
+
+ 	  npx ember-cli new todomvc
+
+
+
+This creates an application development environment that is ready for production and includes the following features by default:
+
+* 	Live reloading development server.
+* 	Third-party packages can richly enrich your application thanks to the plugin architecture.
+* 	Babel and Webpack integration for the newest JavaScript.
+* 	Automated testing environment that allows you to test like a user by running your tests in the browser.
+* 	CSS and JavaScript transpilation and minification for a production build.
+* 	The use of conventions to reduce the variations across applications (allowing easier mental context switching).
+	
+### Installing the shared assets for TodoMVC projects
+
+Installing shared assets, like we're going to do, isn't usually a compulsory step for new projects, but it does allow us to leverage existing shared CSS, which eliminates the need to guess what CSS is required to construct the TodoMVC styles.
```

Comment:
What are shared assets? Why do we need to install them? How is it going to help us?

Reply:
Why are we doing this?